site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -6,11 +6,11 @@ description: >
   Agones provides an mTLS based allocator service that is accessible from outside the cluster using a load balancer. The service is deployed and scales independent to Agones controller.
 ---
 
-To allocate a game server, Agones in addition to {{< ghlink href="pkg/apis/allocation/v1/gameserverallocation.go" >}}GameServerAllocations{{< /ghlink >}}, provides a gRPC service with mTLS authentication, called `agones-allocator`.
+To allocate a game server, Agones in addition to {{< ghlink href="pkg/apis/allocation/v1/gameserverallocation.go" >}}GameServerAllocations{{< /ghlink >}}, provides a gRPC and REST service with mTLS authentication, called `agones-allocator`.
```

Comment:
Our documentation gets updates on each commit to master, so we'll need to wrap the new documentation in `{{% feature %}}` shortcodes for the next release `1.11.0`, so it only shows up on the next release. 

You can see details here:
https://agones.dev/site/docs/contribute/#within-a-page

It may be easiest to take a copy of the existing doc and have two large `feature` sections, one for the old, and one for the new.

Reply:
I have wrapped the updates docs in the `{{feature}}` I think it might be easier to review the changes this way? But happy to copy the whole doc to have the old and new version if you think that it is better that way?

Reply:
This looks good :+1: